trackman/library/forms.py

```diff
@@ -60,3 +62,27 @@ class BulkEditForm(FlaskForm):
         validators=[validators.Optional(), validators.UUID()],
         widget=BootstrapTextInput(),
     )
+
+
+class ArtistMusicbrainzAlbumForm(FlaskForm):
```

Comment:
is this not used?

Reply:
It's used as a subform referenced in ArtistMusicbrainzForm. This is necessary so we can get appropriate album/release group mappings, since we have a variable-length list. Would adding a comment help make this more clear?

Reply:
yes. please add a comment, as it's subtle (I missed it until you mentioned it). Also provide a little better of a description of what these forms do, because the names are unweildy, but not very descriptive (since this is about matching release groups).